sections/06_discussion.md

```diff
@@ -198,6 +198,124 @@ influence healthcare decisions? Or, is deep learning a hypothesis generation
 machine that requires manual validation? DeepChem and DragoNN are worth
 discussing here.*
 
-### Transfer learning/transferability of features
+### Multimodal and transfer learning
 
 * https://github.com/greenelab/deep-review/issues/139#issuecomment-268901804
+
+As discussed above, the fact that biomedical datasets often contain a limited
+number of instances or labels, often leads to poor performance of
+machine learning algorithms. When trained on such datasets, deep learning
+models are particularly prone to overfitting due to their high representational
+power. However, Transfer Learning techniques also known as Domain Adaptation
+enable transfer of extracted patterns between different datasets and even
+domains. This allows a model to take advantage on larger labeled data for
+training of a model which can  then can be reused for the problem in hand. Deep
```

Comment:
Probably this should include the idea that the model is trained for a related problem & reused for the problem at hand. This is implied but it would be good to explicitly state it.